packages/opencensus-web-instrumentation-zone/test/test-interaction-tracker.ts

```diff
@@ -308,6 +316,22 @@ describe('InteractionTracker', () => {
       expect(childSpan.attributes[ATTRIBUTE_HTTP_STATUS_CODE]).toBe('200');
       expect(childSpan.attributes[ATTRIBUTE_HTTP_METHOD]).toBe('GET');
       expect(childSpan.ended).toBeTruthy();
+      const childSpanperformanceEntries = getXhrPerfomanceData(
+        urlRequest,
+        childSpan
+      );
+      expect(childSpanperformanceEntries).toBeTruthy();
+      if (childSpanperformanceEntries) {
+        expect(childSpan.annotations).toEqual(
+          annotationsForPerfTimeFields(
```

Comment:
What would you think about just listing out the hard-coded annotations here? That way we are checking the values directly rather than relying on the correctness of `annotationsForPerfTimeFields`, which in principle could have a bug in both the test and the actual code

This applies to below cases as well